src/main/groovy/life/qbic/datamodel/people/Address.groovy

```diff
@@ -4,25 +4,25 @@ import com.fasterxml.jackson.annotation.JsonProperty
 
 class Address  {
 
-  private String affiliation
-  private String street
-  private Integer zipCode
-  private String country
+  private String affiliation = ""
+  private String street = ""
+  private Integer zipCode = 0
+  private String country = ""
 
   /**
    * Get affiliation
    * @return affiliation
    **/
   @JsonProperty("affiliation")
-  public String getAffiliation() {
+  String getAffiliation() {
     return affiliation;
   }
 
-  public void setAffiliation(String affiliation) {
+  void setAffiliation(String affiliation) {
     this.affiliation = affiliation;
   }
 
-  public Address affiliation(String affiliation) {
+  Address affiliation(String affiliation) {
```

Comment:
This looks wrong.

Reply:
```suggestion
  Affiliation affiliation(String affiliation) {
```
Still relevant.

Reply:
No, it is correct. It is how fluent APIs have been designed for mutable objects. 

And the return type must not change, as it would break the API contract.

Reply:
Ah so it is like a Builder pattern without a builder?

Reply:
old fashioned way, yes.